owslib/wps.py

```diff
@@ -1430,6 +1430,9 @@ def retrieveData(self, username=None, password=None, headers=None, verify=True,
                         headers=headers, verify=verify, cert=cert)
             # extract output filepath from URL query string
             self.fileName = spliturl[1].split('=')[1]
+        elif url.startswith("file://"):
+            u = open(url[7:])
```

Comment:
Would this result in an open file handle ?  Would adding a context manager help here (this block would need some logic rework) https://github.com/geopython/OWSLib/blob/c0d83476e91cf7ef0ec82134ffb828f95685ad06/owslib/wps.py#L1426-L1444)

Reply:
Not sure. Let's not take any chance.